packages/bech32/src/decode.ts

```diff
@@ -0,0 +1,12 @@
+import { BECH32_ADDRESS_LENGTH } from './penumbra-bech32';
+import { bech32m } from 'bech32';
+
+export const bech32ToAddress143 = (addr: string): { inner: Uint8Array } => {
+  const decodeAddress = bech32m.decode(addr, BECH32_ADDRESS_LENGTH);
+  return { inner: new Uint8Array(bech32m.fromWords(decodeAddress.words)) };
+};
+
+export const bech32ToIdentityKey143 = (key: string): { ik: Uint8Array } => {
```

Comment:
Everywhere this is used, this is placed within a new IdentityKey type. Can we change this to return just a `IdentityKey`?

Reply:
ok, i went the other direction. in the interest of minimizing dependency i removed the imported type and now just construct a type that satisfies but is not explicitly an Address or IdentityKey